task/progress.go

```diff
@@ -50,8 +52,10 @@ func ReportProgress(ctx context.Context, lapi *livepeerAPI.Client, taskID string
 	}
 }
 
-func calcProgress(count, size uint64) (val float64) {
+func calcProgress(count, size uint64, startPercentage int, endPercentage int) (val float64) {
 	val = float64(count) / float64(size)
+	val = val * (float64(endPercentage) - float64(startPercentage))
+	val = (val + float64(startPercentage)) / 100
```

Comment:
Yeah I think the math here will be simpler with `0-1` floats as well. Sth like this?

```suggestion
	val = startPercentage + val * (endPercentage - startPercentage)
```